src/components/AvailableParkingSlotsView.tsx

```diff
@@ -41,15 +49,42 @@ export const AvailableParkingSlotsView: React.FC<IAvailableSlotsViewProps> = ({
     );
   }
 
```

Comment:
Declare this below function using an arrow function

Reply:
Declared